### DISCUSSION TITLE
icon.png is not being added to apple pass.

### DISCUSSION THREAD

**Discussion by akash-86** [alexandercerutti/passkit-generator#190] [Q&A]
Hey, I have gone through your pass visualisation website to check the available pass designs. I noticed a small icon on the bottom left side of the pass design. However, when I try to use any icon by setting the icon URL during pass generation, the icon does not appear or append to the pass. This is how I am setting the URL for the icon to "icon.png" -

```javascript
newPass.addBuffer("icon.png", bufferIcon).
```

Is there anything specific that I might be missing here, or is there any concern related to the icon resolution?
I am using event ticket and for that I want to generate a pass in which the icon will be visible on bottom left corner so how is it possible ?
<img width="261" alt="Screenshot 2024-01-30 at 1 24 07 PM" src="https://github.com/alexandercerutti/passkit-generator/assets/143791354/0ce12dbb-f6e8-4ad4-9b27-8189d06deb06">

**Comment by alexandercerutti** [comment]
Hey @akash-86, I've moved the discussion to pkvd repo's, I haven't deleted it! :D

https://github.com/alexandercerutti/passkit-visual-designer/discussions/24

I'm collecting all the information I can in order to answer your the best I can, and answer you there

**Comment by akash-86** [comment]
Sure, no problem but can you please check for this ASAP :D
I have to give an update to my team till tomorrow on this .

**Reply by alexandercerutti** [reply]
Hey, its weekend for me too, dude! And these are open source projects.

If you or your company need support, you can either wait for me, look at the codes and attempt to fix it by your self (by opening a PR) or **help financing the project** (so then I'll give priority over other things in my life).

**Comment by akash-86** [comment]
I also don't work in any company. We are college students and for the
project basis I am using passkit-generator. I appreciate your efforts that
you have provided on this project.
That is why I was just asking if you could help me on this thing.

**Reply by alexandercerutti** [reply]
Sorry if I assumed you were working for a company, but as you can imagine, the situation above is more common that you might expect. I still have to say that your "hurried request" pissed me off a little bit. I hope you understand.

**Comment by akash-86** [comment]
Sorry from my side if it feels like rude, as I am in hurry cuz the project
which was provided from my college is basically of 1 week and now its going
to be end. And I am stuck in the last scenario. No problem let's see what
should be done now.

**Reply by alexandercerutti** [reply]
I don't know if you saw, but I answered you on the other thread
